scripts/gateway.service

```diff
@@ -0,0 +1,35 @@
+# This is a SystemD unit file for the Gateway
+# To configure:
+# - Update the user and group that the service will run as (User & Group below)
+# - Ensure that the gateway binary is in /usr/local/bin and is named gateway (or edit the ExecStart line
+#   below to reflect the name and location of your binary
+# - Ensure that you've run setup and have edited the configuration appropriately prior to starting the
+#   service with this script or adjust the --config-dir option in the ExecSart line
+# To use:
+# - Place this file in /etc/systemd/system/ or wherever your SystemD unit files are stored
+# - Run systemctl daemon-reload
+# - To start run systemctl start gateway
+
+[Unit]
+Description  = Gateway service
+After        = syslog.target network.target
+Wants        = network-online.target
+
+[Service]
+User         = storj
+Group        = storj
+ExecStart    = /usr/local/bin/gateway run --config-dir=/home/storj/.local/share/storj/gateway
```

Comment:
Maybe move this around to be in more normal system locations:

* move binary to /usr/bin/gateway
* move config to /etc/storj/gateway
* include instructions for creating the storj user as a proper system user (without a shell or home dir)

Reply:
I guess the later is easy. for the former i would like to be cautios due to file permissions on the config file. I'll see what i can add to the comments above.